user/user-tool/tool/src/webapp/vm/user/chef_users_edit.vm

```diff
@@ -138,7 +138,7 @@ function removeOptionalAttributeBlock(elem) {
 				<label for="user_pw0">
 					$tlang.getString("usecre.vernewpass")
 				</label>
-				<input type="password" name="pw0" id="user_pw0" onblur="USER.verifyPasswordsMatch();" autocomplete="off" />
+				<input type="password" name="pw0" id="user_pw0" onkeyup="USER.verifyPasswordsMatch()" onblur="USER.verifyPasswordsMatch();" autocomplete="off" />
```

Comment:
Isn't this going to result in users getting the message saying passwords don't match while they are typing into the second box?